flight/goto/mission/main.py

```diff
@@ -0,0 +1,74 @@
+"""
+Main driver code for moving drone to each waypoint
+"""
+
+import asyncio
+import json_parsing
+import goto
+from mavsdk import System
+import mavsdk as sdk
+import logging
```

Comment:
For each of your print statements in main, replace the print() with logging.info() to formalize the logging information, rather than printing it to the console.